typing/ctype.ml

```diff
@@ -2229,14 +2240,16 @@ let rec estimate_type_jkind ~(expand_component : type_expr -> open_type_expr) en
      let is_open, tys_modalities =
        List.fold_left_map
          (fun is_open1 (_lbl, ty) ->
-            let { ty; is_open = is_open2 } = expand_component ty in
-            (is_open1 || is_open2), (ty, Mode.Modality.Value.Const.id))
+            let ({ ty; is_open = is_open2 }, modality) = expand_component ty in
+            (is_open1 || is_open2), (ty, modality))
          false ltys
      in
      (* CR layouts v2.8: This pretty ridiculous use of [estimate_type_jkind]
         just to throw most of it away will go away once we get [layout_of]. *)
      let jkinds =
-       List.map (fun (ty, _) -> estimate_type_jkind ~expand_component env ty)
+       List.map (fun (ty,
+                      (* CR aspsmith: use this modality? *)
```

Comment:
I wasn't able to find anything that failed due to us throwing this away. Plausibly this is just fine to ignore?